app/controllers/users_controller.rb

```diff
@@ -37,6 +37,19 @@ def update
     end
   end
 
+  def destroy
+    if current_user
+      @user = User.find(params[:id])
+      @user.tips.destroy 
```

Comment:
aka "dependent destroy"
http://guides.rubyonrails.org/association_basics.html#options-for-belongs-to